controllers/csi/daemonset.go

```diff
@@ -203,6 +203,20 @@ func prepareDriverSpec(operatorImage string) v1.Container {
 				},
 			},
 		},
+		ReadinessProbe: &v1.Probe{
+			FailureThreshold:    3,
+			InitialDelaySeconds: 5,
+			PeriodSeconds:       5,
+			SuccessThreshold:    1,
+			TimeoutSeconds:      1,
+			Handler: v1.Handler{
+				HTTPGet: &v1.HTTPGetAction{
+					Path:   "/healthz",
```

Comment:
For the sake of consistency, I would also add a [`manager.AddReadyzCheck`](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager) to `cmd/csidriver.main.go`.

Reply:
Done, please double check @baichinger

Reply:
The default endpoint for readiness probe is `/readyz`.